### DISCUSSION TITLE
AR Vertical placement - Supported devices

### DISCUSSION THREAD

**Discussion by felipe-chamas** [google/model-viewer#1803] [Q&A] [answered]
According to [this PR](https://github.com/google/model-viewer/pull/1714), we managed to use vertical AR placement on Android, but since there's no documentation on **Scene Viewer** yet, do you have any information about a list of devices that can handle it?
Another strange behavior occurs on **Quick Look**'s vertical placement, we can do it with any model, allegedly on any device with **iOS > 13**, but on my **iPhone 7 plus** it doesn't work.
Does anybody has any information about the matter to share?

**Comment by elalish** [comment, marked as answer]
I don't believe vertical placement is device-specific on Android, though you probably need to have your apps up to date to have the functionality. I don't know anything about Quick Look's particulars.

**Reply by felipe-chamas** [reply]
All right, thanks for the update! Since we are here, is there another option to this android placement? Anything similar to `vertical_only` or `vertical_preferred` props, or only the one you have used?

**Comment by elalish** [comment]
For WebXR it's one of the other. For SceneViewer, floor is floor only, wall is wall or floor.

**Reply by felipe-chamas** [reply]
Great, thanks for the clarification!
